config/routes.rb

```diff
@@ -1,5 +1,23 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'about', to: 'pages#about', as: :about
+  get 'dashboard', to: 'pages#dashboard', as: :dashboard
```

Comment:
Ah nevermind, I see here we're using the pages controller for the dashboard stuff. That seems fine.